spec/features/editor/deleting_account_spec.rb

```diff
@@ -2,13 +2,14 @@
 
```

Comment:
How about registered_user, to distinguish them from users who are just browsing Pageflows without an account?